### PR TITLE
test for WFCORE-570 Provide ability to display module dependencies vi…

### PR DESCRIPTION
**Pull request by simkam**
…a Management API

https://issues.jboss.org/browse/WFCORE-570

### CODE REVIEW

File: testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/ModuleInfoTestCase.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        for (DependencySpec d : xmlDependencies) {
+            if (d instanceof ModuleDependencySpec) {
+                ModuleDependencySpec mds = (ModuleDependencySpec) d;
+                moduleDependencies.put(mds.getIdentifier().getName() + ":" + mds.getIdentifier().getSlot(), mds);
+            }
+        }
+        int foundDependencies = 0;
+        for (ModelNode d : dependencies) {
+            String dependencyName = d.get("module-name").asString();
+            Assert.assertNotNull("Expected dependency", moduleDependencies.get(dependencyName));
```
Comment (jamezp) [inline testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/ModuleInfoTestCase.java]: This should probably just use `Assert.assertTrue("Expected dependency", moduleDependencies.containsKey(dependencyName));`.

File: testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/ModuleInfoTestCase.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        for (ModelNode n : resourceLoaders) {
+            if (n.get("type").asString().equals("org.jboss.modules.JarFileResourceLoader")) {
+                for (ModelNode node : n.get("paths").asList()) {
+                    paths.add(node.asString());
+                }
+                break;
+            }
+        }
+        String xmlLocalPaths = module.getClassLoader().getLocalPaths().toString();
+        Assert.assertEquals(xmlLocalPaths, paths.toString());
```
Comment (jamezp) [inline testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/ModuleInfoTestCase.java]: This should be `Assert.assertTrue(module.getClassLoader().getLocalPaths().containsAll(paths));`.

**Comment by simkam** [comment]
changed Asserts and rebased

**Comment by simkam** [comment]
retest this please

**Comment by ctomc** [comment]
failures not related to this, there was a faulty component upgrade that caused it.
if your rebase this again it will work.

**Pull request merged**